soap/soap11_endpoint.bal

```diff
@@ -23,37 +23,34 @@ public type Soap11Client client object {
 
     private http:Client soap11Client;
 
-    public function __init(string url, SoapConfiguration? soapConfig = ()) {
-        self.soap11Client = new(url, config = soapConfig.clientConfig);
+    public function __init(string url, http:ClientEndpointConfig? config = ()) {
+        self.soap11Client = new(url, config = config);
     }
 
     # Sends SOAP 1.1 request and expects a response.
     #
-    # + path - Resource path
     # + soapAction - SOAP action
     # + body - SOAP payload
     # + options - SOAP options. Ex: Headers, Ws-addressing parameters, usernameToken parameters
     # + return - If a success, returns the response object, else returns `SoapError` object
-    public remote function sendReceive(string path, string soapAction, xml body, Options? options = ())
+    public remote function sendReceive(string soapAction, xml body, Options? options = ())
             returns SoapResponse|error {
-        return sendReceive(path, soapAction = soapAction, body, options = options, self.soap11Client, SOAP11);
+        return sendReceive(soapAction = soapAction, body, options = options, self.soap11Client, SOAP11);
     }
 
     # Send Robust SOAP 1.1 requests.Sends the request and possibly receives an error.
     #
-    # + path - Resource path
     # + options - SOAP options. Ex: Headers, Ws-addressing parameters, usernameToken parameters
     # + return - If a success, returns `nil`, else returns `SoapError` object
```

Comment:
Docs need to be updated right? There is no `SoapError` object now? Several other occurrences too.